backend/app/services/rag/index/indexer.py

```
@@ -154,22 +217,228 @@ def index_from_binary(
+                        MarkdownNodeParser(
+                            include_metadata=True,
+                            include_prev_next_rel=True,
+                        ),
+                        # Stage 2: Further split large chunks by sentence
+                        SentenceSplitter(
+                            chunk_size=1024,
+                            chunk_overlap=50,
+                        ),
+                    ]
+                )
+                subtype = "markdown_pipeline"
```

Ruff (0.14.14)

[error] Local variable `subtype` is assigned to but never used

Remove assignment to unused variable `subtype`

(F841)